tinygrad/ops.py

```diff
@@ -173,6 +173,8 @@ def __init__(self, buffer: Type[RawBuffer], linearizer_opts, renderer, runtime,
   def to_program(self, k):
     k.linearize()
     src = self.renderer(k.function_name, k.uops)
+    if len(src) == 4:
```

Comment:
You should be able to get the global/local sizes from the linearizer

Reply:
okay, I realized I don't need to invert global but local still needs to be overriden